### PR TITLE
added set_kth_bit.cpp

### PR DESCRIPTION
**Pull request by aman2000raj**
#### Description of Change

Given a number N and a value K. From the right, set the Kth bit in the binary representation of N. The position of Least Significant Bit(or last bit) is 0, the second last bit is 1 and so on.

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [x] Added tests and example, test must pass
- [x] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:  Added an algorithm which set kth bit of a given number.

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C-Plus-Plus/pull/1863"></a>

### CODE REVIEW

**Review by Qannaf — APPROVED** [review]

**Comment by aman2000raj** [comment]
@Panquesito7 please review my PR.

**Comment by aman2000raj** [comment]
@Panquesito7 please review my PR.

File: bit_manipulation/set_kth_bit.cpp [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ * @file
+ * @brief Implementation to [From the right, set the Kth bit in the binary
+ * representation of N]
+ * (https://practice.geeksforgeeks.org/problems/set-kth-bit3724/1/) in an
+ * integer.
+ *
+ * @details
+ *Given a number N and a value K. From the right, set the Kth bit in the binary
+ *representation of N. The position of Least Significant Bit(or last bit) is 0,
+ *the second last bit is 1 and so on. in it.
```
Comment (Panquesito7) [inline bit_manipulation/set_kth_bit.cpp:11]:
```suggestion
 * Given a number N and a value K. From the right, set the Kth bit in the binary
 * representation of N. The position of Least Significant Bit(or last bit) is 0,
 * the second last bit is 1 and so on. in it.
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — COMMENTED** [review]
It's just to address this, and you're done! 🎉

**Review by Panquesito7 — APPROVED** [review]
Awesome work! You've done amazing work, and it's been your
first contribution here! We hope you keep contributing! 😄👍🎉

After this PR is merged, you can ask for the
contributor role in our [**Discord server**](https://discord.gg/c7MnfGFGa6)! 🚀

**Review by mishraabhinn — APPROVED** [review]

**Pull request merged**